modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1314,4 +1337,35 @@ definition {
 		AppBuilderAdmin.assertNewButtonIsVisible(buttonValue = "New Table View");
 	}
 
+	@description = "Validate all fields are available as columns and can be inserted in a Table View"
+	@priority = "5"
+	test ValidateFieldsAddedAsColumns {
+		property portal.acceptance = "false";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.setName(formViewName = "Form View Title - All Fields");
+
+		for (var fieldType : list "Text Field,Select from List,Single Selection,Multiple Selection,Date,Numeric,color-field-type-label,Grid,separator-field-type-label,Upload") {
+			FormViewBuilder.addFieldByDoubleClick(fieldType = "${fieldType}");
+
+			FormViewBuilder.closeSidebar();
+		}
+
+		FormViewBuilder.save();
+
+		AppBuilderAdmin.switchTab(tabName = "Table Views");
+
+		AppBuilderAdmin.add();
+
+		TableViewBuilder.setName(tableViewName = "Table View Title - All Fields");
+
+		for (var fieldType : list "Text Field,Select from List,Single Selection,Multiple Selection,Date,Numeric,color-field-type-label,Grid,separator-field-type-label,Upload") {
+			TableViewBuilder.addColumnByDoubleClick(fieldType = "${fieldType}");
+		}
```

Comment:
Hey @tainguyen100 , could you please add a few more validations here? I would suggest you to Save the Table View and then open it again to see if all the added fields/columns are still there... to check the persistence.